WcaOnRails/config/routes.rb

```diff
@@ -185,7 +185,8 @@
   get 'contact' => 'static_pages#contact'
   get 'documents' => 'static_pages#documents'
   get 'education' => 'static_pages#education'
-  get 'delegates' => 'static_pages#delegates'
+  get 'delegates' => 'application#react_component', defaults: { component_name: "Delegates", title: 'delegates_page.title' }
```

Comment:
Use a specific controller action please. (Also see my other comment above).
This `defaults` and `locals` approach feels way to hacky for something that we haven't even fully decided on yet.

Reply:
Done.